docs/content-onboarding/working-with-GitHub.md

```diff
@@ -61,6 +61,8 @@ Please follow [PC3](../content-ops/PC3.md) for commits and merge requests.
 
 <b>The title of MR MUST contain a definition of the current problem ("Problem: ..."). The proposed solution ("Solution: ...") MUST be written in the description field of MR.</b>
 
+<b>Important Note: Merging is only done by supervisors!</b>
```

Comment:
How does this comments help the student?
How does he know what that implies?

Please think about your problems with this point and add specific instructions. Thx